python/packages/autogen-ext/src/autogen_ext/models/openai/_openai_client.py

```diff
@@ -635,7 +635,6 @@ async def create_stream(
         json_output: Optional[bool] = None,
         extra_create_args: Mapping[str, Any] = {},
         cancellation_token: Optional[CancellationToken] = None,
-        max_consecutive_empty_chunk_tolerance: int = 0,
```

Comment:
Instead of removing this, add deprecated warning in the method body to signal deprecation.